Microsoft.Toolkit.Uwp.UI.Controls/HeaderedItemsControl/HeaderedItemsControl.cs

```diff
@@ -0,0 +1,331 @@
+﻿using System.Diagnostics;
+using Microsoft.Toolkit.Uwp.UI.Controls.Data;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Data;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Represents a control that contains multiple items and has a header.
+    /// </summary>
+    [StyleTypedProperty(Property = "ItemContainerStyle", StyleTargetType = typeof(ContentPresenter))]
+    public partial class HeaderedItemsControl : ItemsControl
```

Comment:
Sorry, I can't find a WinRT control like HeaderedItemsControl. Which one are you referring?